asciidoc/client-applications.adoc

```diff
@@ -90,37 +94,48 @@ include::{javascript-examples}/examples.test.js[tags=driver-lifecycle]
 // ----
 ======
 
-// [.include-with-python]
-// ======
-// [source, python]
-// ----
-// include::{python-examples}/driver_lifecycle_example.py[tags=driver-lifecycle]
-// ----
-// ======
+[.include-with-python]
+======
+[source, python]
+----
+include::{python-examples}/test_driver_lifecycle_example.py[tags=driver-lifecycle]
```

Comment:
these example snippets are not working. As discussed on Slack, the location has changed: 
from `neo4j-python-driver/tree/1.7/test/examples`
to `neo4j-python-driver/tree/4.0/tests/integration/examples`